.github/workflows/python-app.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
```

Comment:
this might need a branch selector to work

Reply:
I copy-pasted from here and it worked there: https://github.com/Segelzwerg/FamilyFoto/blob/master/.github/workflows/python-check.yml

Reply:
So currently it will only run on pushes to main, but on PR to all branches?

Reply:
Exactly